pyproject.toml

```diff
@@ -166,12 +168,12 @@ dependencies = [
 
 [project.optional-dependencies]
 llm = [
-    "transformers",
-    "torch",
     "langchain<=0.0.301", # Weirdly fails with newer version, see https://linear.app/giskard/issue/GSK-1853/updating-lanchain-version-over-00301-weirdly-fails
-    "evaluate",
-    "bert-score",
-    "datasets"
+    "rapidfuzz",          # needed for garak
```

Comment:
we should get rid of `rapidfuzz`, it's not needed anymore.

Reply:
Done!